composer.json

```diff
@@ -20,37 +20,9 @@
         "symfony/phpunit-bridge": "^4.0"
     },
     "scripts": {
-        "fast-install" : [
-            "php scripts/check.php",
-            "php bin/console doctrine:database:create --if-not-exists",
-            "php bin/bin/doctrine dbal:import claroline.sql -vvv",
-            "php bin/console assets:install public --symlink",
-            "php scripts/data-symlink.php"
-        ],
-        "sync": [
-            "composer fetch",
-            "composer build",
-            "php bin/console claroline:update --env=prod",
-            "composer disable-maintenance"
-        ],
-        "sync-dev": [
-            "composer fetch-dev",
-            "composer build",
-            "php bin/console claroline:update -vvv",
-            "composer disable-maintenance"
-        ],
-        "fetch": [
-            "composer install --prefer-dist --no-dev --optimize-autoloader",
-            "npm install"
-        ],
-        "fetch-dev": [
-            "composer install --prefer-source",
-            "npm install"
-        ],
         "build": [
             "Claroline\\CoreBundle\\Library\\Installation\\Refresher::deleteCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "npm run webpack"
```

Comment:
Do we no longer need to run "npm run webpack"?
If we need it, then shall we add it to the steps for a clean installation from source?

Reply:
You need it for prod env only - for dev it's `npm run webpack:dev` - that's why I've deleted it from the `build`. I need to update the doc. It will be done in : https://github.com/claroline/Claroline/pull/1510.

But it's possible we don't need the build cmd anymore.